python_modules/dagster/dagster/_core/definitions/decorators/asset_decorator.py

```diff
@@ -130,8 +136,8 @@ def asset(
             contains letters, numbers, and _) and may not contain python reserved keywords.
         ins (Optional[Mapping[str, AssetIn]]): A dictionary that maps input names to information
             about the input.
-        non_argument_deps (Optional[Union[Set[AssetKey], Set[str]]]): Set of asset keys that are
-            upstream dependencies, but do not pass an input to the asset.
+        deps (Optional[Sequence[Union[AssetsDefinition, SourceAsset, CoercibleToAssetKey]]]):
+             Set of asset keys that are upstream dependencies, but do not pass an input to the asset.
```

Comment:
alright - i didnt understand when i first commented, but I now understand not using `CoercibleToAssetKey` in the API docs since its an internal-ish type alias